ports/stm/boards/stm32f746g_disco/mpconfigboard.mk

```diff
@@ -0,0 +1,14 @@
+USB_VID = 0x239A
+USB_PID = 0x809C
```

Comment:
I figure this is new since it doesn't match the Nucleo F767 or the F746, but I figured I'd double check.

Reply:
Yeah, I got them quite some time ago from ladyada.